src/NuGet.Core/NuGet.Protocol.Core.v2/V2Utilities.cs

```diff
@@ -44,6 +43,12 @@ public static IPackageRepository GetV2SourceRepository(Configuration.PackageSour
                         if (httpReq != null)
                         {
                             httpReq.UserAgent = _userAgent;
+
+                            if (httpReq.Headers.Get(AcceptLanguage.HeaderName)?.Length == 0 
```

Comment:
Couldn't `httpReq.Headers.Get(AcceptLanguage.HeaderName)?.Length == 0` more simply be `string.IsNullOrEmpty(httpReq.Headers.Get(AcceptLanguage.HeaderName))`? What's the intent?